lib/Cleantalk/Common/API.php

```diff
@@ -1050,10 +1050,6 @@ public static function checkResponse($result, $method_name = null)
      */
     public static function sendLocalSettings($settings)
```

Comment:
Please, make it similar to other methods in this class.

Use self::sendRequest($params, $url); instead of Helper::httpRequest()
use self::checkResponse() before return the value.